src/groups/bmq/bmqt/bmqt_uri.t.cpp

```diff
@@ -657,6 +657,28 @@ static void test7_testLongUri()
     bmqt::UriParser::shutdown();
 }
 
+static void test8_testUriParserOverflow()
+{
+    s_ignoreCheckDefAlloc = true;
+    // Disable the default allocator check. When 'bmqt::Uri'
+    // is created from a long uri string error message is
+    // logged via 'BALL_LOG_ERROR' which allocates using
+    // the default allocator.
+
+    mwctst::TestHelper::printTestName("URI PARSER OVERFLOW TEST");
+
+    // Initialize bmqt::UriParser many times, and then deinitialize
+    // it the same number of times according to the usage contract.
+    const bsls::Types::Int64 k_INIT_NUM = 2200000000LL;
```

Comment:
Would `static_cast<int64_t>( numeric_limits<int32_t>::max() ) + 1` (or corresponding BDE-ism) be clearer?  It's a somewhat lower number of `initialize`/`shutdown`s, but says more clearly what we're trying to test.

Reply:
I removed this test from this branch, because it was too slow, 43 seconds on my M2.